aten/src/ATen/core/TensorImpl.h

```diff
@@ -904,9 +904,9 @@ struct CAFFE2_API TensorImpl : public c10::intrusive_ptr_target {
    * a tensor on CPU and then CopyFrom a CUDA tensor, that will to a
    * CUDA-to-CPU transfer).
    *
-   * If the function is invoked without `context` the copy would be synchronous
+   * 'async' parameter triggers async copy for CUDA tensors
    */
-  void CopyFrom(const TensorImpl& src, at::BaseContext* context = nullptr) {
+  void CopyFrom(const TensorImpl& src, bool async = false) {
```

Comment:
Nit: In PyTorch, we call this `non_blocking`, because `async` is a reserved keyword in Python so we can't use it as a keyword argument. Just a thing to be aware of :)